src/server.js

```diff
@@ -2,10 +2,20 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const {spawn} = require('child_process');
 const app = express();
-const port = 80;
+const port = 8080;
+let debug = false;
+let env = Object.create( process.env );
 app.use(bodyParser.json());
 app.use(bodyParser.text());
 
+const checkDebug = (req) => {
+    console.log(req.query.debug)
+    if(req.query.debug === undefined)
+        return false;
+    env.TEST_MODE = true;
```

Comment:
I'm not sure we want to use TEST_MODE here. We should use some other variable. TEST_MODE actually has a pretty specific meaning related to testing using cmake (it removes any platform/machine specific results from the output so the test cases work on any machine).

Reply:
Makes sense. You mentioned something like `IS_DOCKER=true`. Perhaps that is better?

Reply:
I implemented IS_DOCKER.